metadata-ingestion/src/datahub/ingestion/run/pipeline.py

```diff
@@ -56,11 +56,13 @@ def __init__(self, config: PipelineConfig):
         self.ctx = PipelineContext(run_id=self.config.run_id)
 
         source_type = self.config.source.type
+        logger.info(source_type)
```

Comment:
ignore these, I had a bug in the config / create section and was getting None response.